### PR TITLE
Setup updatecli for oblt-cli version

### PR DESCRIPTION
**Pull request by reakaleek**

### CODE REVIEW

File: .ci/updatecli/values.d/scm.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+scm:
+  enabled: true
+  owner: elastic
+  repository: oblt-actions
+branch: main
```
Comment (v1v) [inline .ci/updatecli/values.d/scm.yml:5]: typo?

```suggestion
  branch: main
```

File: update-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Config file for `updatecli compose ...`.
+# https://www.updatecli.io/docs/core/compose/
+policies:
+  - name: Handle default oblt-cli version
+    policy: ghcr.io/elastic/oblt-updatecli-policies/oblt-cli/version:0.0.2@sha256:dd79537aa64a28c57da351182a70a2a798e9b6c5eb55c243fdbd4f5bffe5133d
+    values:
+      - .ci/updatecli/values.d/scm.yml
+      - .ci/updatecli/values.d/oblt-cli.yml
```
Comment (v1v) [inline update-compose.yaml:8]: Can you also add support for:

- https://github.com/elastic/apm-agent-php/pull/1193

**Review by v1v — COMMENTED** [review]
You need to configure [this](https://github.com/orgs/elastic/packages/container/oblt-updatecli-policies%2Foblt-cli%2Fversion/settings) with the list of Repositories that can use it.

File: update-compose.yaml (hunk 2 above)
Comment (reakaleek) [reply update-compose.yaml:8]: https://github.com/elastic/oblt-actions/pull/83/commits/31103f512408ae5d0df29908028fb0337a405d42

**Review by v1v — APPROVED** [review]
LGTM, just a minor comment to use the default folder

**Comment by reakaleek** [comment]
> LGTM, just a minor comment to use the default folder

Sorry, where is the comment? 😅

**Comment by reakaleek** [comment]
⬆️ @v1v

**Comment by v1v** [comment]
> ⬆️ @v1v

Ups, I was expecting my comment, maybe I didn't type enter 😢 

I was saying that we can use the default updatecli folder structure:
- https://github.com/elastic/apm-pipeline-library/tree/main/updatecli
- https://github.com/updatecli/udash/tree/main/updatecli/values.d

and so on. 

It's not what we do in almost all the projects, since `.ci` was already in place. But we can apply the default for the projects we own, if we prefer

**Comment by reakaleek** [comment]
Okay. thanks!

> > ⬆️ @v1v
> 
> Ups, I was expecting my comment, maybe I didn't type enter 😢
> 
> I was saying that we can use the default updatecli folder structure:
> 
> * https://github.com/elastic/apm-pipeline-library/tree/main/updatecli
> * https://github.com/updatecli/udash/tree/main/updatecli/values.d
> 
> and so on.
> 
> It's not what we do in almost all the projects, since `.ci` was already in place. But we can apply the default for the projects we own, if we prefer

Okay thanks.. will definitely do in a follow-up.

**Comment by reakaleek** [comment]
> Okay. thanks!
> 
> > > ⬆️ @v1v
> > 
> > 
> > Ups, I was expecting my comment, maybe I didn't type enter 😢
> > I was saying that we can use the default updatecli folder structure:
> > 
> > * https://github.com/elastic/apm-pipeline-library/tree/main/updatecli
> > * https://github.com/updatecli/udash/tree/main/updatecli/values.d
> > 
> > and so on.
> > It's not what we do in almost all the projects, since `.ci` was already in place. But we can apply the default for the projects we own, if we prefer
> 
> Okay thanks.. will definitely do in a follow-up.

@v1v I'm actually afraid this could be confusing in this repo.

the updatecli folder is the tool action folder. I'm afraid that it could be confused to be part of an action? WDYT?

**Pull request merged**